core/templates/pages/exploration-player-page/services/extract-image-filenames-from-model.service.ts

```diff
@@ -118,6 +118,9 @@ export class ExtractImageFilenamesFromModelService {
   _getSolutionHtml(state: State): string {
     let languageCode = (
       this.contentTranslationLanguageService.getCurrentContentLanguageCode());
+    if (state.interaction.solution === null) {
+      return '';
+    }
```

Comment:
The state.interaction.solution variable threw 'Object is possibly null error'. So to prevent this error there is a check to see that if solution is null, which means that the solution in the state does not exist. If it is null then function returns null.  I used an empty string instead of a null in the previous commit because it was throwing errors, but was able to replace it and make it work with a null check.